README.md

```diff
@@ -84,6 +84,23 @@ download and unarchive the source tarball (Appium-Python-Client-X.X.tar.gz).
     python setup.py install
     ```
 
+## Compatibility Matrix
+
+|Appium Python Client| Selenium binding|
+|----|----|
+|`2.10.0`+ |`4.1.0`+ |
```

Comment:
should it be 4.9.1+ ?

Reply:
it worked with older versions, so currently this section does not limit the version. If we need to limit the selenium version, the setup.py will also need to limit it